### PR TITLE
PTS-24475: App: MSD: Delete key does not delete the focused chip

### PR DESCRIPTION
**Pull request by mescalantecommon**
[App: MSD: Delete key does not delete the focused chip](https://cawiki.atlassian.net/browse/PTS-24475)
When the user press DEL or Backspace on a chip, delete the target chip and focus the following elements:

- If there are still chips on the left side, focus the previous chip
- If there are not chips on the left side, select the first, that is the one on the right side of the deleted chip.
- If there are not more chips, focus on the MSD.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,110 @@
 # Changelog
 
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
```
Comment (geebru) [inline CHANGELOG.md]: This changelog is huge for the fix you're working on. Where did all these items come from?

Additionally, we should only be bumping versions _during_ the release workflow and not as part of feature work. Please reset the package/lock files back to 5.0.0 and we can bump during the creation/merge of the release branch.

**Comment by geebru** [comment]
@mescalantecommon Changes look good aside from my comment. The release workflow we follow here is similar to the one used within Forge: https://cawiki.atlassian.net/wiki/spaces/CA/pages/3232366595/Development+workflow#Releases

Once these changes look good and are merged:
1. We'll create a new branch from `develop` for the release
2. In that branch we'll version bump and update the changelog
3. Once approved, we'll merge that branch into `master` and `develop`
4. Once both are merged, we pull `master`, run builds, run tests, create a new Git tag, and run the npm publish command.

I can help with that as it's not a flow most here are used to, but that's the steps here. We're lacking a lot of automation here as we don't interact very often with ng-select or its repo so if there's any confusion, let me know!

**Comment by mescalantecommon** [comment]
> @mescalantecommon Changes look good aside from my comment. The release workflow we follow here is similar to the one used within Forge: https://cawiki.atlassian.net/wiki/spaces/CA/pages/3232366595/Development+workflow#Releases
> 
> Once these changes look good and are merged:
> 
> 1. We'll create a new branch from `develop` for the release
> 2. In that branch we'll version bump and update the changelog
> 3. Once approved, we'll merge that branch into `master` and `develop`
> 4. Once both are merged, we pull `master`, run builds, run tests, create a new Git tag, and run the npm publish command.
> 
> I can help with that as it's not a flow most here are used to, but that's the steps here. We're lacking a lot of automation here as we don't interact very often with ng-select or its repo so if there's any confusion, let me know!

Thank you for you comments @geebru!
Question regarding this PR, can I merge it to develop as it is, or should I remove the other file changes `(CHANGELOG.md` and `package.json`) and just keep the `ng-select.components.ts` changes?

**Comment by geebru** [comment]
> Thank you for you comments @geebru! Question regarding this PR, can I merge it to develop as it is, or should I remove the other file changes `(CHANGELOG.md` and `package.json`) and just keep the `ng-select.components.ts` changes?

Changes should be made first to keep the branch focused on the changes made. Changelog and package will be updated in the release branch 👍🏻

**Review by geebru — CHANGES_REQUESTED** [review]
Officially requesting changes, which I realize I had not done :)

File: CHANGELOG.md (hunk 1 above)
Comment (mescalantecommon) [reply CHANGELOG.md]: I've wrongly run a command `node ../../node_modules/standard-version/bin/cli.js --infile ../../CHANGELOG.md` and I didn't realize the change until was pushed. I'll revert this chenges

Comment (geebru) [reply CHANGELOG.md]: @mescalantecommon The original author has a setup that auto-generates and runs changelogs so no worries. It's a great tool but we don't work in a manner that helps it run well :P

**Review by geebru — APPROVED** [review]
Changes look good!

**Pull request merged**